src/main/java/frc/robot/Utilities/SwerveAlignment.java

```diff
@@ -0,0 +1,106 @@
+        frontLeftAlignmentDisplayRad = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("FL Test Offset (r)",frontLeftInitialAngle - m_dDrivetrain.getFrontLeftAngle() - DriveConstants.kFrontLeftOffset).withPosition(1,1).withWidget(BuiltInWidgets.kTextView).getEntry();
+        //Front right
+        frontRightInitialAngle = m_dDrivetrain.getFrontRightAngle();
+        frontRightAngleDisplay = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("Front Right Angle",m_dDrivetrain.getFrontLeftAngle()).withPosition(3,0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        frontRightRawAngleDisplay = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("FR Raw Angle",m_dDrivetrain.getFrontRightAngle() - DriveConstants.kFrontRightOffset).withPosition(4,0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        frontRightOffsetDisplay = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("Front Right Offset",DriveConstants.kFrontRightOffset).withPosition(5,0).withSize(1,2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        frontRightAlignmentDisplayDeg = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("FR Test Offset (d)",Math.toDegrees(frontRightInitialAngle - m_dDrivetrain.getFrontRightAngle() - DriveConstants.kFrontRightOffset)).withPosition(3,1).withWidget(BuiltInWidgets.kTextView).getEntry();
+        frontRightAlignmentDisplayRad = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("FR Test Offset (r)",frontRightInitialAngle - m_dDrivetrain.getFrontRightAngle() - DriveConstants.kFrontRightOffset).withPosition(4,1).withWidget(BuiltInWidgets.kTextView).getEntry();
+        //Back left
+        backLeftInitialAngle = m_dDrivetrain.getBackLeftAngle();
+        backLeftAngleDisplay = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("Back Left Angle",m_dDrivetrain.getBackLeftAngle()).withPosition(0,2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        backLeftRawAngleDisplay = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("BL Raw Angle",m_dDrivetrain.getBackLeftAngle() - DriveConstants.kBackLeftOffset).withPosition(1,2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        backLeftOffsetDisplay = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("Back Left Offset",DriveConstants.kBackLeftOffset).withPosition(2,2).withSize(1,2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        backLeftAlignmentDisplayDeg = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("BL Test Offset (d)",Math.toDegrees(backLeftInitialAngle - m_dDrivetrain.getBackLeftAngle() - DriveConstants.kBackLeftOffset)).withPosition(0,3).withWidget(BuiltInWidgets.kTextView).getEntry();
+        backLeftAlignmentDisplayRad = Shuffleboard.getTab(SWERVE_ALIGNMENT).add("BL Test Offset (r)",backLeftInitialAngle - m_dDrivetrain.getBackLeftAngle() - DriveConstants.kBackLeftOffset).withPosition(1,3).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
Based on the widget being added in line 69 I cannon tell what "BL Test Offset (r)" is if I were to not know anything about the code and I would recommend rewording the widget to be "BL Test Offset_rad"